Can't build examples/hysterisis.cpp for d1_mini

I'm getting an error for the examples/hysterisis.cpp when building for d1_mini, BUT not for esp32. I don't understand why it happens. 

Error message:
In file included from src\hysteresis.cpp:7:0:
lib\SensESP_ci\src/transforms/hysteresis.h: In instantiation of 'Hysteresis<IN, OUT>::Hysteresis(IN, IN, OUT, OUT, String) [with IN = float; OUT = bool]':
src\hysteresis.cpp:42:72:   required from here
lib\SensESP_ci\src/transforms/hysteresis.h:30:67: error: invalid initializer for array member 'const ParamInfo Hysteresis<float, bool>::param_info [4]'
             low_output, high_output, this->param_info, config_path) {}
                                                                   ^

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -10,23 +10,23 @@ jobs:
       matrix:
         example: [
           examples/ads1x15_volt_meter.cpp,
-          examples/ina219_example.cpp,
           examples/analog_input.cpp,
-          examples/lambda_transform.cpp,
           examples/bme280_example.cpp,
           examples/bmp280_example.cpp,
+          examples/fuel_flow_meter.cpp,
+          examples/fuel_level_sensor/fuel_level_sensor_example.cpp,
+          examples/gps_compass.cpp,
+          examples/hysteresis.cpp,
+          examples/ina219_example.cpp,
+          examples/lambda_transform.cpp,
+          examples/milone_level_sensor/milone_level_sensor.cpp,
+          examples/onewire_temperature/onewire_temperature_example.cpp,
           examples/relay_control.cpp,
+          examples/rpm_counter.cpp,
           examples/sht31_example.cpp,
           examples/temperature_sender.cpp,
-          examples/gps_compass.cpp,
-          examples/fuel_flow_meter.cpp,
-          examples/rpm_counter.cpp,
-          examples/fuel_level_sensor/fuel_level_sensor_example.cpp,
-          examples/onewire_temperature/onewire_temperature_example.cpp,
-          examples/milone_level_sensor/milone_level_sensor.cpp,
           examples/thermocouple_temperature_sensor/max31856_thermocouple_example.cpp,
           examples/ultrasonic_level_sensor/ultrasonic_dvp_example.cpp]
-          # examples/hysteresis.cpp,
     steps:
     - uses: actions/checkout@v1
     - name: Set up Python

--- a/src/transforms/hysteresis.h
+++ b/src/transforms/hysteresis.h
@@ -3,14 +3,21 @@
 
 #include "lambda_transform.h"
 
+// Web UI parameter name definition
+const ParamInfo hysteresis_param_info[4] = {
+    {"lower_threshold", "Lower threshold"},
+    {"upper_threshold", "Upper threshold"},
+    {"low_output", "Low output"},
+    {"high_output", "High output"}};
+
 /**
  * @brief Hysteresis function
- * 
+ *
  * Hysteresis is a threshold switch with a dead zone: to prevent switching
  * back and forth at the threshold value, input has to rise above the
  * upper threshold before being switched high, and has to decrease below
  * the lower threshold before being switched back low.
- * 
+ *
  * @tparam IN Input variable type
  * @tparam OUT Output variable type
  * @param[in] lower_threshold Lower threshold value
@@ -27,15 +34,9 @@ class Hysteresis : public LambdaTransform<IN, OUT, IN, IN, OUT, OUT> {
              OUT high_output, String config_path = "")
       : LambdaTransform<IN, OUT, IN, IN, OUT, OUT>(
             &(Hysteresis::function), lower_threshold, upper_threshold,
-            low_output, high_output, this->param_info, config_path) {}
+            low_output, high_output, hysteresis_param_info, config_path) {}
 
  private:
-  // Web UI parameter name definition
-  const ParamInfo param_info[4] = {{"lower_threshold", "Lower threshold"},
-                                   {"upper_threshold", "Upper threshold"},
-                                   {"low_output", "Low output"},
-                                   {"high_output", "High output"}};
-
   // Function implementing the actual hysteresis logic
   static OUT function(IN input, IN lower_threshold, IN upper_threshold,
                       OUT low_output, OUT high_output) {

